Uninett/nav
# Fix related `modules` in device history search

When trying to filter device history by module due to the change in related names in #2418 this leads to 
```
FieldError at /devicehistory/

Cannot resolve keyword 'module' into field.
```

How to reproduce: 
1. Go to `/devicehistory/`.
2. Under `Module` select a module and click on `View module history`.
3. See error. 

This fixes this by replacing `module` with `modules` at the relevant places.
